relay-general/src/pii/convert.rs

```diff
@@ -186,6 +221,34 @@ THd+9FBxiHLGXNKhG/FRSyREXEt+NyYIf/0cyByc9tNksat794ddUqnLOg0vwSkv
         })
     }
 
+    fn sensitive_cookies() -> serde_json::Value {
+        serde_json::json!({
+        // Common session cookie names for popular web frameworks
+        "sentrysid": "my sentrysid", // Sentry default session cookie name
+        "sudo": "my sudo",   // Sentry default sudo cookie name
+        "su": "my su",    // Sentry superuser cookie name
+        "session": "my session",
+        "sessionid": "my sessionid",
+        "user_session": "my user_session",
+        "symfony": "my symfony",
+        "phpsessid": "my phpsessid",
+        "fasthttpsessionid": "my fasthttpsessionid",
+        "mysession": "my mysession",
+        "irissessionid": "my irissessionid",
+        // Common CSRF/XSRF cookie names for popular web frameworks
+        "csrf": "my csrf",
+        "xsrf": "my xsrf",
+        "_xsrf": "my _xsrf",
+        "_csrf": "my _csrf",
+        "csrf-token": "my csrf-token",
+        "csrf_token": "my csrf_token",
+        "xsrf-token": "my  xsrf-token",
+        "xsrf_token": "my  xsrf_token",
+        "fastcsrf": "my fastcsrf",
+        "_iris_csrf": "my _iris_csrf",
```

Comment:
Let's add one non-sensitive entry here, to test that it is _not_ scrubbed, e.g.

```suggestion
        "_iris_csrf": "my _iris_csrf",
        "not_sensitive": "keep this",
```